arena-artifacts/values.yaml

```diff
@@ -28,8 +28,8 @@ binary:
 # tf-operator
 tf:
   enabled: true
-  image: acs/tf_operator
```

Comment:
I suggest using another PR to update the image tag version.

Reply:
@cheyang I have already raised a separated PR to update the tf-operator image, see #1200.